backends/edk2-compat/edk2-svc-read.c

```diff
@@ -600,8 +600,8 @@ static int getSizeFromSizeFile(size_t *returnSize, const char* path)
 	close(fptr);
 	// turn string into base 10 int
 	*returnSize = strtol(c, NULL, 0); 
-	// strol likes to return zero if there is no conversion from string to int
-	// so we need to differentiate an error from a file that actually contains 0
+	//strol likes to return zero if there is no conversion from string to int
+	//so we need to differentiate an error from a file that actually contains 0
```

Comment:
I see this seems to be a pattern. I'd recommend doing a treewide search for this and making it consistent. (`grep -re "//\w"` may help)

Reply:
Saving these for `clang-format` PR